packages/smooth_app/lib/generic_lib/dialogs/smooth_alert_dialog.dart

```diff
@@ -35,12 +35,46 @@ class SmoothAlertDialog extends StatelessWidget {
       shape: const RoundedRectangleBorder(
         borderRadius: ROUNDED_BORDER_RADIUS,
       ),
-      content: Column(
-        mainAxisSize: MainAxisSize.min,
-        children: <Widget>[
-          _buildTitle(context),
-          SizedBox(height: height, child: body),
-        ],
+      content: ConstrainedBox(
+        constraints: BoxConstraints(maxHeight: height ?? double.infinity),
```

Comment:
Did you check if all appearances of SmoothAlertDialog pass a height and if not so, if they looks somewhat okay with `double.infinity` also what about using `double.infinity * 0.5` or something like this as a max instead